### PR TITLE
Done

### PR DESCRIPTION
**Pull request by DevSpaciX**

### CODE REVIEW

File: cinema/urls.py [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
+        "get": "retrieve",
+        "put": "update",
+        "patch": "partial_update",
+        "delete": "destroy",
+    }
+)
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```
Comment (ArsenPidhoretskyi) [inline cinema/urls.py:30]: No need to include if basepath not given
urlpatterns = [list of paths] + router.urls

File: cinema/views.py [hunk 2]
```diff
@@ -1,45 +1,107 @@
...
+    mixins.ListModelMixin, mixins.CreateModelMixin, generics.GenericAPIView
+):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        return self.create(request, *args, **kwargs)
```
Comment (ArsenPidhoretskyi) [inline cinema/views.py:68]: I think all will works if you remove this methods

File: cinema/views.py [hunk 3]
```diff
@@ -1,45 +1,107 @@
...
+        return self.retrieve(request, *args, **kwargs)
+
+    def put(self, request, *args, **kwargs):
+        return self.update(request, *args, **kwargs)
+
+    def patch(self, request, *args, **kwargs):
+        return self.partial_update(request)
+
+    def delete(self, request, *args, **kwargs):
+        return self.destroy(request, *args, **kwargs)
```
Comment (ArsenPidhoretskyi) [inline cinema/views.py:90]: I think all will works if you remove this code

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: cinema/views.py (hunk 2 above)
Comment (olena-hul) [reply cinema/views.py:68]: Remove methods please

File: cinema/views.py (hunk 3 above)
Comment (olena-hul) [reply cinema/views.py:90]: And here

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Comment by DevSpaciX** [comment]
image
image
Here full describing : https://mate-parttime.slack.com/archives/C0427SK8V4H/p1676820067652919

**Pull request closed without merging**